pubber/src/main/java/daq/pubber/Pubber.java

```diff
@@ -73,6 +74,8 @@ public class Pubber {
 
   public static final int SCAN_DURATION_SEC = 10;
   public static final String DISCOVERY_ID = "RANDOM_ID";
+  public static final String IOT_ENDPOINT_BLOB_NAME = "_iot_config";
```

Comment:
Move to a constant somewhere - where's a good spot that won't be overridden by gencode?